### ISSUE TITLE
Not connect Ledger hardware wallet

### ISSUE THREAD

**Issue by probojnik** [MetaMask/metamask-extension#10249]
OS
MacOS 10.13.6 (17G12034)

Browser:
Brave 1.19.86 Chromium: 88.0.4324.96

Hardware:
Ledger Nano S
firmware version 
Secure element 1.6.1
mikrocontroller 1.12
bootloader 0.11

Error:
When connecting, following the instructions exactly, after a long spinner I get the following error: 
We had trouble connecting to your ledger, try reviewing our hardware wallet connection guide and try again.

Could you tell me how to fix it?

**Comment by darkwing** [comment]
Thank you for the report @probojnik!  Did your Ledger suddenly not connect or have you always had this problem?

**Comment by probojnik** [comment]
I haven't tried using Metamask before.

**Comment by Lindgrenium** [comment]
> Thank you for the report @probojnik! Did your Ledger suddenly not connect or have you always had this problem?

I have same problem:

OS
MacOS 11.0.1 (20B29)

Browser:
Firefox 84.0.2 (64-bit)

Hardware:
Ledger Nano X
firmware version 1.2.4-5, updated from 1.2.4-1

**Comment by probojnik** [comment]
Any update ?

**Comment by tmashuang** [comment]
Is ledger live installed? Does the [ledger connection debugging](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues) for Mac solve your issue?

**Comment by zaraaah** [comment]
Hi, I'm having the same issue but with Nano X - it takes me to the spinning wheel and no authorization prompt comes up on my ledger and transaction doesn't go through. Any help?

**Comment by alexwykoff** [comment]
I can reproduce on my Mac as well.

Using a Ledger Nano S,
MacOS Big Sur 11.2.2
Chrome 89.0.4389.72
Metamask 9.1.1

The Ledger was attached to Metamask, but then seemed to have connection issues. I removed it then try to re-add it. 
In the inspector I see an error after a failed connection :
```
TransportError: Failed to sign with Ledger device: U2F DEVICE_INELIGIBLE
  | (anonymous) | @ | initSentry.js:8
-- | -- | -- | --
  | (anonymous) | @ | ui.js:152
```

Ledger Live app 2.23.0 connects to the Nano S without issue. 

Happy to provide more diagnostics if given guidance. Also glad to hop on a call if needed.

**Comment by darkwing** [comment]
Hello @alexwykoff @zaraaah @probojnik Oftentimes when this error is seen, it's because the Ledger device is either (1) not unlocked or (2) the Ethereum app is not opened on the device.  Please ensure it's unlocked and the Ethereum app is open, and let us know if that works!

**Comment by alexwykoff** [comment]
Thanks @darkwing!

Indeed, I was able to reconnect once I had the Ethereum app open on the Ledger.

**Comment by darkwing** [comment]
Cheers @alexwykoff !

**Comment by oliviertr** [comment]
This one is a very strange bug, in that whether or not it connects seems to be completely random. Sometimes it connects, and sometimes, mysteriously, it doesn't. I haven't been able to pinpoint a reason why and I've tried a lot of different things.

**Comment by nocibambi** [comment]
I have the same issue. However, it worked on another Ubuntu (18.04) with Firefox.

Ubuntu 22.04
Firefox 105.0.1 (64-bit)
Metamask: 10.18.4

Ledger Nano S Plus
Firmware: 1.0.4
Secure Element: 1.0.4
Microcontroller: 4.03
Bootloader: 3.12

Ledger Live 2.48.0
Ledger Live Ethereum App: 1.9.19

Ledger Live works without any problems. Though, when I start it, it produces the following errors (I have [Appimage Launcher](https://appimage.org/) installed and causes some of it:

```shell
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
AppImageLauncher error: appimage_shall_not_be_integrated() failed (returned -1)
AppImageLauncher error: appimage_is_terminal_app() failed (returned -1)

(AppImageLauncher:132767): GLib-CRITICAL **: 16:21:13.989: g_file_test: assertion 'filename != NULL' failed
Ledger Live 2.48.0
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[133235:0929/162118.923990:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
spawned internal process 133597
init
Checking for update
Update for version 2.48.0 is not available (latest version: 2.48.0, downgrade is disallowed).
```
